curiouslearning/FeedTheMonsterJS
# fix: next button issue reloading (FM-340)

# Changes
- Fix next button in levelend scene reloading

# How to test
- Open the start screen of FTM.
- Tap or click any part of the start screen to proceed to the next screen
- Select any level
- Play the game and finish
- Make sure next button is showing if more than or equal to 2 stars
- Make sure next button is hidden when less than 2 stars
- Make sure next button is hidden when last level

Ref: [FM-340](https://curiouslearning.atlassian.net/browse/FM-340)


[FM-340]: https://curiouslearning.atlassian.net/browse/FM-340?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
